Dockerfile.worker

```diff
@@ -20,7 +20,9 @@ EXPOSE 8080
 COPY --from=builder /etc_extra/ /etc/
 USER 65534
 COPY --from=builder --chown=nobody /etc/ssl/certs/ca-certificates.crt /etc/ssl/certs/
+COPY --from=builder --chown=nobody /tmp /tmp
```

Comment:
whats in the tmp in the builder?

Reply:
It is the "standard" tmp folder, which is used by the worker to process things (in enterprise).